src/index.ts

```diff
@@ -48,6 +69,21 @@ const getLogin = async (octokit: Octokit): Promise<string> => {
   return login;
 }
 
+// If repoConfig (the file repositories.json) exists, read it and return the list of repositories from the repositories key
+let repoConfig = 'repositories.json'
```

Comment:
`grse-config.json` might be a better name?

The structure should probably not just be `Repository[]`, as otherwise you can't specific your desired preferences in future. Maybe it being an object containing a `repos: Repository[]` entry, so it is extensible with the options you want to set etc. (i.e. the `Preferences` type).